Included makefile and packaging resources for build

bgphipps opened the pull request:
@sheeley @mgloystein Quick and easy makefile for installing resources and combining lua files. 

We can/should include tests in the `Makefile` when we're at that point. 

`pack.lua` was a small snippet I found for combining files. I'm open to `squish` ( @mgloystein suggested it) if we feel we need more capabilities rather than just combining files.

bgphipps committed:
included makefile and packaging resources for build

mgloystein commented:
Coolio

sheeley commented on Makefile:
```diff
@@ -0,0 +1,9 @@
+	lua buildresources/pack.lua src/ > dist/planout.lua
+
+install:
+	brew install lua
```
Might want to note that this is OS X specific, and/or support other things? Also would be useful to include a README.md explaining how this (and other things) work.

mgloystein replied:
Any version of make should work.

sheeley commented on Makefile:
```diff
@@ -0,0 +1,9 @@
+	luarocks install penlight
+
+clean:
+	rm dist/planout.lua
```
Adding `-f` will keep this from complaining if the file doesn't exist.

bgphipps replied:
I'll update it.

sheeley commented on buildresources/pack.lua:
```diff
@@ -0,0 +1,71 @@
+-- pack.lua
+-- released under the Romantic WTF Public License
+-- http://getmoai.com/wiki/index.php?title=User:Pygy/Romantic_WTF_Public_License
```
great license

bgphipps replied:
yeah, it's pretty silly.

bgphipps commented:
@sheeley @mgloystein per our conversation, I removed `dist/planout.lua` and removed installing `lua` in `make install`

I'll work on circleci integration in a different pr.
